packages/supabase/src/utils/generateFilter.ts

```diff
@@ -35,6 +44,11 @@ export const generateFilter = (filter: CrudFilter, query: any) => {
             item.operator !== "and" &&
             "field" in item
           ) {
+            if (item.operator === "ina" || item.operator === "nina")
+              return `${item.field}.${mapOperator(
+                item.operator,
+              )}.${`{${item.value.map((val: any) => `"${val}"`).join(",")}}`}`;
+
             return `${item.field}.${mapOperator(item.operator)}.${item.value}`;
```

Comment:
I think it's more readable

```suggestion
            let value = item.value;

            if (item.operator === "ina" || item.operator === "nina") {
              value = `{${item.value.map((val: any) => `"${val}"`).join(",")}}`;
            }

            return `${item.field}.${mapOperator(item.operator)}.${value}`;
```